boards/shields/lcd_par_s035/boards/frdm_mcxn947_mcxn947_cpu0.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2024, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+&flexio_i2c {
+	status = "okay";
+	gt911_lcd_par_s035: gt911-lcd_par_s035@5d {
+		compatible = "goodix,gt911";
```

Comment:
No need to duplicate the full definition here, you can just delete the `reset-gpios` property and add `alt-addr`, right?

Reply:
Hello Daniel, 

I think that these commit 
b38a4ccbdbc     Dominik Lau     Fri Sep 27 10:17:05 2024 +0200  boards: added touch controller to the `/chosen` node
f8f70c2ac72     Daniel DeGrasse Mon Aug 26 15:58:32 2024 +0000  boards: shields: lcd_par_s035: enable reset gpio within input module
3604aba75b1     Daniel DeGrasse Mon Sep 23 09:29:43 2024 -0500  shields: lcd_par_s035: rename nxp_flexio_lcd to zephyr_mipi_dbi_parallel

already introduce the touch on S035 LCD so this PR might not be very relevant anymore

I tested on 4.0.0 tag 
west build  -b frdm_mcxn947/mcxn947/cpu0  zephyr/samples/subsys/input/input_dump  -- -DSHIELD=lcd_par_s035_8080

input_dump works OK
Input sample started
I: input event: dev=gt911-lcd_par_s035@5d     type= 3 code=  0 value=27
I: input event: dev=gt911-lcd_par_s035@5d     type= 3 code=  1 value=28

But lvgl demo 

west build  -b frdm_mcxn947/mcxn947/cpu0  zephyr/samples/subsys/display/lvgl   -- -DSHIELD=lcd_par_s035_8080

has a button that is not clickable

I suspect, as we had tried the prio trick too, that when both display and touch are used it does not 
work as espected.
Could you please test and confirm, to rule out I am doing something stupid ? :)

Thnaks !!

g.

Reply:
Locally when I build and flash `samples/subsys/display/lvgl` on 4.0.0 like so: `west build -p always -b frdm_mcxn947/mcxn947/cpu0 samples/subsys/display/lvgl --shield=lcd_par_s035_8080`, I see a button and can click it.

Maybe check what cmake prints for your Zephyr version?
```
Zephyr version: 4.0.0 (/home/danieldegrasse/zephyr/zephyrproject/zephyr)
```

is what I see.

Reply:
Hello Daniel!

I changed display and it indeed works :)

Thnaks so much

I think we can close this PR here